contributing-to/reference-architectures.md

```diff
@@ -2,9 +2,9 @@
 
 ## What is a Reference Architecture?
 
-The term *reference architecture* means a lot of things, but in the context of the Azure Architecture Center, it refers to a specific type of guidance artifact, found [here](https://docs.microsoft.com/azure/architecture/reference-architectures/). 
+The term *reference architecture* means a lot of things, but in the context of the Azure Architecture Center, it refers to a specific type of guidance artifact, found [here](/azure/architecture/reference-architectures/).
```

Comment:
This file doesn't get published so we need the full URL here

Reply:
Ah, that makes sense. Fixed.